.github/workflows/linters.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.eslintrc.json
           [ -f .babelrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.babelrc
       - name: ESLint Report
-        run: npx eslint .
+        run: npx eslint .  --fix
```

Comment:
Kindly do not edit linters file. If you need to fix some linters errors, you should just run the command locally. We are not allow to edit linters files. Kindly address that. 👍🏾